templates/lgi/brcm972180hbc-refboard-TR0003667.json

```diff
@@ -0,0 +1,202 @@
+{
+    "platformName": "brcm972180hbc-refboard-TR0003667",
+    "os": "linux",
+    "arch": {
+        "arch": "arm",
+        "variant": "v7"
+    },
+    "rdk": {
+        "version": "2020Q4",
+        "supportedFeatures": [
+            "Controller",
+            "DeviceInfo",
+            "Monitor",
+            "OCDM",
+            "TraceControl",
+            "WifiManager",
+            "com.comcast.CoPilot",
+            "com.comcast.DeviceProvisioning",
+            "com.comcast.FrameRate",
+            "com.comcast.HdcpProfile",
+            "com.comcast.HdmiInput",
+            "com.comcast.StateObserver",
+            "com.comcast.StorageManager",
+            "org.rdk.ActivityMonitor",
+            "org.rdk.DeviceDiagnostics",
+            "org.rdk.DisplaySettings",
+            "org.rdk.FrontPanel",
+            "org.rdk.HomeNetworking",
+            "org.rdk.LoggingPreferences",
+            "org.rdk.Network",
+            "org.rdk.OCIContainer",
+            "org.rdk.Proxies",
+            "org.rdk.SleepTimer",
+            "org.rdk.TTSResource",
+            "org.rdk.TTSSettings",
+            "org.rdk.Timer",
+            "org.rdk.UserPreferences",
+            "org.rdk.Warehouse"
+        ]
+    },
+    "hardware": {
+        "graphics": true
+    },
+    "storage": {
+        "persistent": {
+            "storageDir": "/opt/dac_apps/data/0",
+            "maxSize": "100M"
+        }
+    },
+    "gpu": {
+        "extraMounts": [
+            {
+                "source": "/usr/share/X11/xkb",
+                "destination": "/usr/share/X11/xkb",
+                "type": "bind",
+                "options": [
+                    "rbind",
+                    "nosuid",
+                    "nodev",
+                    "ro",
+                    "X-mount.mkdir"
+                ]
+            },
+            {
+                "source": "/tmp/nxserver_ipc",
+                "destination": "/tmp/nxserver_ipc",
+                "type": "bind",
+                "options": [
+                    "bind",
+                    "ro"
+                ]
+            },
+            {
+                "source": "/etc/ssl/certs",
+                "destination": "/etc/ssl/certs",
+                "type": "bind",
+                "options": [
+                    "rbind",
+                    "nosuid",
+                    "nodev",
+                    "ro",
+                    "X-mount.mkdir"
+                ]
+            },
+            {
+                "source": "/usr/share/ca-certificates",
+                "destination": "/usr/share/ca-certificates",
+                "type": "bind",
+                "options": [
+                    "rbind",
+                    "nosuid",
+                    "nodev",
+                    "ro",
+                    "X-mount.mkdir"
+                ]
+            },
+            {
+                "source": "/tmp/rialto-0",
```

Comment:
Shouldn't this be "source": "/tmp/{id}" ? With fixed src /tmp/rialto-0 you will always have to name your dac app "rialto-0" when installing it?? Don't want to block this merge. You can fix later.

Reply:
Unless you are still using rialto-servermanager-sim, in that case it is indeed always src /tmp/rialto-0. But starting with the rdkshell-rialto integration it doesn't need to be fixed anymore.

Reply:
FYI I have a pull request (draft) where I update some of my templates to use "source": "/tmp/{id}": https://github.com/rdkcentral/BundleGen/pull/104 to work with the rdkshell-rialto integration

Reply:
@svenko766 please raise another PR to change the source to /tmp/{id}